tools/helper.py

```
@@ -124,3 +125,11 @@ def init_skale(wallet: BaseWallet) -> Skale:
 
 def init_defualt_wallet() -> Skale:
     return RPCWallet(TM_URL)
+
+
+def safe_load_yml(filepath):
+    with open(filepath, 'r') as stream:
+        try:
+            return yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```

print is useless here. Better to add log. Also it's a little strange that we just ignore exception. It may cause problems if we will have logic depending on the content of the file.

---

updated